mne/decoding/transformer.py

```diff
@@ -497,6 +497,10 @@ class FilterEstimator(TransformerMixin):
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
         Defaults to self.verbose.
+
```

Comment:
again keep it for now. @LauriParkkonen has code that uses it and still works on real time decoding.